### ISSUE TITLE
openvidu-react module not found

### ISSUE THREAD

**Issue by rush86999** [OpenVidu/openvidu-call-react#5]
hello i m getting a 
```Can't resolve '/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/assertThisInitialized'```

is this project still being used?

**Comment by pabloFuente** [comment]
Hi,

That path `/home/pablo/Desktop/` is in fact my local path in my development machine, but I can't find it anywhere in OpenVidu repository. It is not commited anywhere, unless I'm missing something.

How can I replicate this issue?

**Comment by rush86999** [comment]
I did an npm install openvidu-react instead of openvidu-call-react, that's how I ended up the with the error.  I checked the folder and seems like a copy of openvidu-call-react with those additions. so I deleted it and copied the code that I think was relevant to work with react and openvidu-browser. I haven't tested anything so not sure if it works.

**Comment by pabloFuente** [comment]
I just run `npm install` over [openvidu-call-react](https://github.com/OpenVidu/openvidu-call-react) project and [openvidu-library-react](https://github.com/OpenVidu/openvidu-tutorials/tree/master/openvidu-library-react) tutorial and they work fine. No strange path anywhere.

**Comment by rush86999** [comment]
i think it was probably an old repository `openvidu-react`.

**Comment by keisen** [comment]
Hi @pabloFuente .
I'm facing the same problem on web-browser now.

```
./node_modules/openvidu-react/dist/components/VideoRoomComponent.js
Module not found: Can't resolve '/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/assertThisInitialized' in '/home/vagrant/workspace/openvidu-tutorials/openvidu-library-react/node_modules/openvidu-react/dist/components'
```

I checked out current (today's) master branch of `openvidu-tutorials` and I performed `npm install` in `./openvidu-tutorials/openvidu-library-react` .
The head of `./node_modules/openvidu-react/dist/components/VideoRoomComponent.js` was below:

```
import _classCallCheck from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/classCallCheck";
import _createClass from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/createClass";
import _possibleConstructorReturn from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/possibleConstructorReturn";
import _getPrototypeOf from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/getPrototypeOf";
import _inherits from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/inherits";
import _assertThisInitialized from "/home/pablo/Desktop/openvidu-call-react/library/node_modules/@babel/runtime/helpers/esm/assertThisInitialized";
```

And then, It's working well when I downgraded `openvidu-react` from `2.7.1` to `2.6.0`.
The latest version seems that have a problem.
I hope this report will be of some help.

Best Regards,
Keisen.

**Comment by pabloFuente** [comment]
Okey, so there was a dependency update related to react libraries that broke the building of openvidu-react package. Roll back to previous versions of react dependencies has been done.

New version 2.7.2 has been released to NPM and 2.7.1 has been deprecated. We have also updated openvidu-library-react tutorial to use this new version. It should work fine now.

Please check it out and post here your results.

Tnx!
